### ISSUE TITLE
link rel / style type need to be include.

### ISSUE THREAD

**Issue by mikedidthis** [rnmp/salvattore#73]
tl;dr I :cry: then I read the source and I :smile:

For salvatorre.js to work, the `<link>` or `<style>` must include the correct tags:

```
<style type="text/css">...</style>
<link rel="stylesheet">...</style>
```

These, however, can be omitted in HTML5. Stylesheet: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/style#Attributes Link: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/link#attr-rel

If these are omitted, salvatorre.js silent fails, as they are required here: https://github.com/rnmp/salvattore/blob/master/dist/salvattore.js#L319

I found this looking at #71 

Maybe an idea to update the documentation to include that requirement.

I saw something about using a custom rel / tag on the css specific to salvattore: #23 

Could we add in something like that, or even:

```
<style type="text/salvattore">...</style>
```

Happy to add in, but I think it needs discussion.
